scripts/preprocess/execute_preprocessing.R

```diff
@@ -0,0 +1,46 @@
+# This is the yaml that orchestrates the steps required for preprocessing the 
```

Comment:
could you just clarify that "This" refers to the file named preprocessing.yaml, where as this execute_preprocessing.R file just describes how to use that file?

Reply:
good point - I'll push a change soon